accepted/00039-content-lifecycle-management.md

```diff
@@ -171,18 +175,44 @@ similar to the schedule definition of a repository sync.
 Adding a timer is optional.
 
 
-## Support restore
+## Restore functionality
+
+Support limited *Content* version tracking and restoring (or rolling back).
+
+The most important entity is the *Snapshot*. It has a version and it contains all the information needed for re-creating a *Build* in particular state in a future:
+* software channels reference (also the pointer which channel is the base (or "leader) in the *Build*
+* packages per channel reference
+* errata per channel
+* others (after we implement support for them): activation key, configuration channels, ...
+
+### Create Content Snapshot (backup)
+*Snapshot* is created after a successful *Project* *build*. It is populated with the references to channels and their packages/errata based on the *build*. Note that the software channels references point to the **source** channels of a project, not to the built ones.
```

Comment:
> Note that the software channels references point to the **source** channels of a project, not to the built ones.

While reading it, I feel I am missing a clarification about it, where *source* means *original*, and *built* means *generated by the current project for the built*. Or something like that.